README.md

```diff
@@ -33,22 +31,71 @@ A Clojure java-client wrapper for Couchbase Server 4. Now updated to support 6.0
          => "{\"hitpoints\":23832,\"level\":141,\"loggedIn\":true,\"name\":\"Aaron1\",\"jsonType\":\"player\",\"experience\":248,\"uuid\":\"78edf902-7dd2-49a4-99b4-1c94ee286a33\"}"
     
     ;; to create/update a json
-    (b/replace bucket "Aaron1" "{
+    (b/replace! bucket "Aaron1" "{
         \"experience\": 248,
         \"hitpoints\": 23832,
         \"jsonType\": \"player\",
         \"level\": 141,
         \"loggedIn\": true,
         \"name\": \"Aaron1\",
         \"uuid\": \"78edf902-7dd2-49a4-99b4-1c94ee286a33\"
+        \"sponsors\": [\"Nike\" \"Reebok\" \"Freddies\"]
+        \"tues\": [{\"symptom\": {\"asthma\": \"Ventolin Inhaler\"}}
+                   {\"symptom\": {\"pain\": \"Asprin\"}}]  
         }")
-        => {:id "Aaron1", :cas 5090205399720, :expiry 0, :mutation-token nil, :content {:hitpoints 23832, :level 141, :loggedIn true, :name "Aaron1", :jsonType "player", :experience 248, :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33"}}
+        => {:id "Aaron1", :cas 1569768233881239552, :expiry 0, :mutation-token nil, :content {:name "Aaron1", :hitpoints 23832, :experience 248, :level 141, :loggedIn true, :tues [{:symptom {:asthma "Ventolin Inhler"}} {:symptom {:pain "Asprin"}}], :sponsors ["Nike" "Reebok" "Freddies"], :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33", :jsonType "player"}}
         
-        (b/replace bucket "Aaron1" {:hitpoints 23832, :level 141, :loggedIn true, :name "Aaron1", :jsonType "player", :experience 248, :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33"})
-            => {:id "Aaron1", :cas 6654687454080, :expiry 0, :mutation-token nil, :content {:hitpoints 23832, :level 141, :loggedIn true, :name "Aaron1", :jsonType "player", :experience 248, :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33"}}
+        (b/replace! bucket "Aaron1" {:name "Aaron1", :hitpoints 23832, :experience 248, :level 141, :loggedIn true, :tues [{:symptom {:asthma "Ventolin Inhler"}} {:symptom {:pain "Asprin"}}], :sponsors ["Nike" "Reebok" "Freddies"], :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33", :jsonType "player"})
+
+        => {:id "Aaron1", :cas 1569768233881239552, :expiry 0, :mutation-token nil, :content {:name "Aaron1", :hitpoints 23832, :experience 248, :level 141, :loggedIn true, :tues [{:symptom {:asthma "Ventolin Inhler"}} {:symptom {:pain "Asprin"}}], :sponsors ["Nike" "Reebok" "Freddies"], :uuid "78edf902-7dd2-49a4-99b4-1c94ee286a33", :jsonType "player"}}
+
+       ;; To lookup particular objects / values in a document, pass a list of paths. Vectors in
+       ;; the response are converted from [n] to <n> as brackets are not valid in keywords.
+       ;; The DocumentFragment used to execute the request also has an .exists method that returns
+       ;; true or false for a given path. This has not been implemented. Instead you will get a
+       ;; {:keyword nil} response for a noneixstent path.
+	  
+       (b/lookup-in bucket "Aaron1" "name" "sponsors" "tues[1]" "injuries")
```

Comment:
maybe this can be an array to make it easier to understand the parameters?
  (b/lookup-in bucket "Aaron1" ["name" "sponsors" "tues[1]" "injuries"])

Reply:
I see what you mean, looking at it after a month, it is not clear is it that "Aaron1" is the key and the rest are JSON paths. Let me see what can be done.